src/board.js

```diff
@@ -0,0 +1,154 @@
+// This is the board as an array
+let board = [
```

Comment:
Avoid global code  ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/global_requirement.png)

Reply:
On it.

Reply:
Actually, the board variable is inside a module, an is private.